.github/workflows/ds-storybook-deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Design System Storybook
+
+on:
+  push:
+    paths:
+      - "packages/design-system/**"
+    branches:
+      - "main"
+
+# Change the working directory for all the jobs in this workflow
+defaults:
+  run:
+    working-directory: packages/design-system
+
+jobs:
+  deploy-storybook:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the code
+      - uses: actions/checkout@v3
+
+      # Setup node with version 16.x and NPM registry url
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+
+      # yarn build to build the project
+      - name: Build package
+        run: yarn build-storybook
+
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::391311798816:role/design-systems-github-actions-deploy-s3
+          role-session-name: ds-storybook-deploy-workflow
+          aws-region: us-east-2
```

Comment:
Shall we move region git secret key?

Reply:
We used to do that for some other older workflows, but, creates more problems than it solves. Also, not technically a secret, so I think we should be okay with it here.